src/components/relatedcompare/CompareModal.jsx

```diff
@@ -1,21 +1,22 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import CompareTable from './CompareTable.jsx'
 
 const CompareModal = ({ product, sliderInfo, currentCompare, openModal, setOpenModal }) => {
-
   const displayModal = (e) => {
     e.preventDefault();
-    setOpenModal(!openModal)
+    setOpenModal(!openModal);
+    document.body.style.overflow = "scroll";
+    console.log(currentCompare)
```

Comment:
Hey looks good but could you remove this console.log?

Reply:
Console.log is removed.